shopify.com.website2.json

```diff
@@ -0,0 +1,26 @@
+{  
+   "providerId":"shopify.com",
+   "providerName":"Shopify",
+   "serviceId":"website2",
```

Comment:
What is the difference to https://github.com/Domain-Connect/Templates/blob/master/shopify.com.website.json ?
Seems like the same template, just variables are named differently.